dcos/provider.go

```diff
@@ -72,13 +72,14 @@ func Provider() terraform.ResourceProvider {
 			"dcos_marathon_pod": resourceDcosMarathonPod(),
 		},
 		DataSourcesMap: map[string]*schema.Resource{
-			"dcos_base_url":        dataSourceDcosBaseURL(),
-			"dcos_job":             dataSourceDcosJob(),
-			"dcos_package_config":  dataSourceDcosPackageConfig(),
-			"dcos_package_version": dataSourceDcosPackageVersion(),
-			"dcos_service":         dataSourceDcosService(),
-			"dcos_token":           dataSourceDcosToken(),
-			"dcos_version":         dataSourceDcosVersion(),
+			"dcos_base_url":               dataSourceDcosBaseURL(),
+			"dcos_job":                    dataSourceDcosJob(),
+			"dcos_package_config":         dataSourceDcosPackageConfig(),
+			"dcos_package_version":        dataSourceDcosPackageVersion(),
+			"dcos_service":                dataSourceDcosService(),
+			"dcos_token":                  dataSourceDcosToken(),
+			"dcos_version":                dataSourceDcosVersion(),
+			"dcos_service_account_secret": dataSourceDcosServiceAccountSecret(),
```

Comment:
`dcos_security_secret_service_account_secret` as of `dcos security secrets create-sa-secret`

Reply:
Yup, that sounds good 👍